File: centerpark-back/src/main/java/model/Aquatique.java
package model;

import java.time.LocalDate;
import java.time.LocalTime;

import javax.persistence.Entity;
@Entity

public class Aquatique extends Activite{

public Aquatique() {
}	
	public Aquatique(Meteo meteo, LocalDate date, LocalTime heure, double prix) {
		super(meteo, date, heure, prix);
	}


	@Override
	public String toString() {
		return "Aquatique [id=" + id + ", meteo=" + meteo + ", date=" + date + ", heure=" + heure + ", prix=" + prix
				+ ", reservations=" + reservations + "]";
	}


}

File: olivier/angular/formationAngular/src/app/notions/model/aquatique.ts
import { Time } from '@angular/common';
import { Meteo } from './meteo';
import { Activite } from './activite';
export class Aquatique extends Activite {
  public constructor(
    id?: number | undefined,
    meteo?: Meteo | undefined,
    date?: Date | undefined,
    heure?: Time | undefined,
    prix?: number | undefined
  ) {
    super(id, meteo, date, heure, prix);
  }
}
